### PR TITLE
Adjust CSS to more closely match the printed book.

### PR DESCRIPTION
**Pull request by gitvsh**
For all content:
  - Fix chapter title texts overlapping in multi-line.
  - Adjust the overall visible width and text aligment.
  - Align bylines to the right.
  - Align images to center.
  - Add border for asides at the start of chapters.

For the TOC:
  - Bold and resize top-level items.
  - Space out each Part.
  - Delete unused CSS from the TOC section.

### CODE REVIEW

File: raw/theme/html/html.css [hunk 1]
```diff
@@ -824,40 +828,37 @@ a.co img { padding: 0; }
...
-  margin-bottom: 8px !important;
-  margin-left: 0px !important;
-  padding-left: 0px !important;
-}
-
-div.toc ol ol {
-  margin-left: 30px !important;
-  padding-left: 0px !important;
+nav[data-type="toc"] li {
+  border: 0px solid #BEBEBE;  /* For debugging, set border non-zero. */
```
Comment (lasagnaboy) [inline raw/theme/html/html.css:832]: Remove before merging? Not sure of "debugging" status in the comment

**Review by lasagnaboy — APPROVED** [review]
One question about the CSS, comment inline

File: raw/theme/html/html.css (hunk 1 above)
Comment (gitvsh) [reply raw/theme/html/html.css:832]: Sure. The goal was to leave a _disabled_ styling, but maybe we'll never need that.

Comment (gitvsh) [reply raw/theme/html/html.css]: Done!

**Pull request merged**